test/tests/mortar/displaced-gap-conductance-2d-non-conforming/gap-conductance.i

```diff
@@ -99,6 +99,16 @@
   [../]
 []
 
+[Materials]
+  [constant]
+    type = GenericConstantMaterial
+    prop_names = 'gap_conductance'
+    prop_values = '.03'
```

Comment:
Logic should work on non-constant Material Properties created internally (e.g. Mortar constraint initialization), right? Where are these projections happening in this PR?

Reply:
I'm not quite sure what you're asking but i evaluate slave and master face material properties [here](https://github.com/idaholab/moose/pull/14872/files#diff-1526b1ec72736a09e386eb5a57012066R162) and [here](https://github.com/idaholab/moose/pull/14872/files#diff-1526b1ec72736a09e386eb5a57012066R191) respectively.